python/ambassador/envoy/v2/v2listener.py

```diff
@@ -811,8 +811,7 @@ def __init__(self, config: 'V2Config', service_port: int) -> None:
             if req_hdrs:
                 self.base_http_config["tracing"]["request_headers_for_tags"] = req_hdrs
 
-        proper_case = self.config.ir.ambassador_module.get('proper_case', False)
```

Comment:
I actually think that this is better left alone -- what're you trying to solve here?

Reply:
the if check failed for me with the .get method, it was always evaluating as false(even after I set it to true in the config).

Reply:
Gotcha, thanks.